phantom-finagle/src/main/scala/com/websudos/phantom/finagle/package.scala

```diff
@@ -0,0 +1,450 @@
+        table = select.table,
+        rowFunc = select.rowFunc,
+        init = select.init,
+        wherePart = select.wherePart,
+        orderPart = select.orderPart,
+        limitedPart = enforceLimit,
+        filteringPart = select.filteringPart,
+        usingPart = select.usingPart,
+        count = select.count,
+        options = select.options
+      ).singleCollect()
+    }
+  }
+
+  implicit class GenericQueryAugmenter[
+    Table <: CassandraTable[Table, _],
+    Record,
+    Limit <: LimitBound,
+    Order <: OrderBound,
+    Status <: ConsistencyBound,
+    Chain <: WhereBound,
+    PS <: HList
+  ](val query: Query[Table, Record, Limit, Order, Status, Chain, PS]) extends AnyVal {
+    def ttl(duration: com.twitter.util.Duration): Query[Table, Record, Limit, Order, Status, Chain, PS] = {
+      query.ttl(duration.inSeconds)
+    }
+  }
+
+  implicit class UpdateQueryAugmenter[
+    Table <: CassandraTable[Table, _],
+    Record,
+    Limit <: LimitBound,
+    Order <: OrderBound,
+    Status <: ConsistencyBound,
+    Chain <: WhereBound,
+    PS <: HList
+  ](val query: UpdateQuery[Table, Record, Limit, Order, Status, Chain, PS]) extends AnyVal {
+
+    def ttl(duration: com.twitter.util.Duration): UpdateQuery[Table, Record, Limit, Order, Status, Chain, PS] = {
+      query.ttl(duration.inSeconds)
+    }
+  }
+
+  implicit class AssigmentsUpdateQueryAugmenter[
+    Table <: CassandraTable[Table, _],
+    Record,
+    Limit <: LimitBound,
+    Order <: OrderBound,
+    Status <: ConsistencyBound,
+    Chain <: WhereBound,
+    PS <: HList,
+    ModifiedPrepared <: HList
+  ](val query: AssignmentsQuery[Table, Record, Limit, Order, Status, Chain, PS, ModifiedPrepared]) extends AnyVal {
+
+    def ttl(duration: Duration): AssignmentsQuery[Table, Record, Limit, Order, Status, Chain, PS, ModifiedPrepared] = {
+      query.ttl(duration.inSeconds)
+    }
+  }
+
+  implicit class ConditionalUpdateQueryAugmenter[
+    Table <: CassandraTable[Table, _],
+    Record,
+    Limit <: LimitBound,
+    Order <: OrderBound,
+    Status <: ConsistencyBound,
+    Chain <: WhereBound,
+    PS <: HList,
+    ModifiedPrepared <: HList
+  ](val query: ConditionalQuery[Table, Record, Limit, Order, Status, Chain, PS, ModifiedPrepared]) extends AnyVal {
+
+    def ttl(duration: Duration): ConditionalQuery[Table, Record, Limit, Order, Status, Chain, PS, ModifiedPrepared] = {
+      query.ttl(duration.inSeconds)
+    }
+  }
+
+  implicit class PreparedSelectQueryAugmenter[
+    Table <: CassandraTable[Table, _],
+    Record,
+    Limit <: LimitBound
+  ](val block : ExecutablePreparedSelectQuery[Table, Record, Limit]) extends AnyVal {
+
+    /**
+      * Get the result of an operation as a Twitter Future.
+      *
+      * @param session The Datastax Cassandra session.
+      * @return A Twitter future wrapping the result.
+      */
+    def get()(implicit session: Session, keySpace: KeySpace, ev: =:=[Limit, Unlimited]): Future[Option[Record]] = {
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Do not prefix getters with 'get'](https://www.codacy.com/app/flavian/phantom/file/2525508486/issues/source?bid=2847375&fileBranchId=3265057#l428)